configurations/pfrich_only.yml

```
@@ -1,4 +1,4 @@
 features:
-- beampipe
-- pid:
-    - pfrich
+  beampipe:
```

```suggestion
  # beampipe:
```
Prefer to not have the beampipe by default, to be consistent with the dRICH. We use these `_only.yml` configurations for standalone RICH simulation tests.

---

Ah nevermind, I have that in another open PR, no need to do it here

---

Sounds good

> Ah nevermind, I have that in another open PR, no need to do it here